sites-enabled/10-www.freecodecamp.org.conf

```diff
@@ -1035,13 +1035,22 @@ server {
   }
 }
 
+# maps Euler Redirect URLs
+map $request_uri $euler_redirects_uri {
+  include /etc/nginx/maps/euler-redirects.map;
+}
+
```

Comment:
I don't think it's a good idea to have a special variable just for euler redirects. It implies that there's some difference between these and the other legacy 301 redirects, but that does not seem to be the case.

Instead, I would just put all the euler redirects inside legacy-302-redirects.map and delete the extra config here.
```suggestion
```